<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LH7U05CG6452K0D" name="test.java.TestUPAEnrollment" tests="1" failures="1" timestamp="09 Mar 2018 10:05:00 GMT" time="82.262" errors="0">
  <testcase name="testSuccessfulAOProviderEnrollment" time="82.262" classname="test.java.TestUPAEnrollment">
    <failure type="java.lang.AssertionError" message="Enrollment Completed successfully is not present on the pageorg.openqa.selenium.NoSuchElementException: Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;//td[contains(text(),&#039;Enrollment Completed Successfully&#039;)]&quot;}
Command duration or timeout: 30.09 seconds">
      <![CDATA[java.lang.AssertionError: Enrollment Completed successfully is not present on the pageorg.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//td[contains(text(),'Enrollment Completed Successfully')]"}
Command duration or timeout: 30.09 seconds
	at org.testng.Assert.fail(Assert.java:93)
	at main.java.reporting.LogTemp.failure(LogTemp.java:86)
	at main.java.reporting.LogTemp.Fail(LogTemp.java:66)
	at main.java.nativeFunctions.Element.verifyElementPresent(Element.java:147)
	at main.java.pageObjects.ThankYouPage.<init>(ThankYouPage.java:28)
	at main.java.pageObjects.ValidateEFTERASubmitInfo.submitForm(ValidateEFTERASubmitInfo.java:92)
	at main.java.pageObjects.ValidateEFTERASubmitInfo.fillEnrollersInfo(ValidateEFTERASubmitInfo.java:85)
	at main.java.pageObjects.UPARegistrationPage.doCompleteEnrollment(UPARegistrationPage.java:167)
	at test.java.TestUPAEnrollment.testSuccessfulAOProviderEnrollment(TestUPAEnrollment.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testSuccessfulAOProviderEnrollment -->
</testsuite> <!-- test.java.TestUPAEnrollment -->
